src/nvim/window.c

```diff
@@ -3074,10 +3076,9 @@ int tabpage_index(tabpage_T *ftp)
  * FAIL.
  * Careful: When OK is returned need to get a new tab page very very soon!
  */
-static int 
-leave_tabpage (
-    buf_T *new_curbuf,        /* what is going to be the new curbuf,
-                                          NULL if unknown */
+static int leave_tabpage(
+    buf_T *new_curbuf,        // what is going to be the new curbuf,
+                              // NULL if unknown
     int trigger_leave_autocmds
 )
```

Comment:
I was stopping myself for doing too much styling but since you propose it. It's better that way